build-system/npm-publish/write-package-files.js

```diff
@@ -165,6 +166,9 @@ async function writePackageJson() {
       react: '^17.0.0',
     },
   };
+  if (useBentoCore) {
```

Comment:
Are we planning on making a version that doesn't require bento core and a version that does?
Would it make sense to always use core?